plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -491,4 +585,19 @@ protected List<InvalidCastTestCase> invalidCast()
                 new InvalidCastTestCase("c_timetz", "int"),
                 new InvalidCastTestCase("c_timetz", "bigint"));
     }
+
+    @Test
+    void testCastPushdownWithCharConvertedToVarchar()
+    {
+        try (TestTable table = new TestTable(
+                getQueryRunner()::execute,
+                TEST_SCHEMA + "." + "case_sensitive_1_",
```

Comment:
Is this test related to case sensitivity?

Reply:
not related, renamed table prefix. Showcases that table with greater than max char precision when created through Trino gets converted to varchar.